mqtt/mqtt-broker-tests-util/src/server.rs

```diff
@@ -77,7 +77,7 @@ where
     P: MakePacketProcessor + Clone + Send + Sync + 'static,
 {
     lazy_static! {
-        static ref PORT: AtomicU32 = AtomicU32::new(5555);
+        static ref PORT: AtomicU32 = AtomicU32::new(8889);
```

Comment:
There is a chance of port collision if you have iotedge runtime running on your dev box. So I bumped it up to a more "safe" range of ports.